### PR TITLE
resurrecting the modal-cover-fullscreen class

### PR DESCRIPTION
**Pull request by rmisio**
Resurrecting the modal-cover-fullscreen css class because it turns out the .modal-opaque.modal-navBlock only covers the Page Nav which results in a non-themed background color.

image

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -197,55 +242,59 @@ $(window).bind('keydown', function(e) {
...
-      case config.keyShortcuts.save:
-        window.obEventBus.trigger('saveCurrentForm');
-        break;
-      case config.keyShortcuts.refresh:
-        Backbone.history.loadUrl();
-        break;
-    }
-
-    if (route !== null) {
+  switch (char) {
```
Comment (jjeffryes) [inline js/main.js]: Is there where @drwasho was saying the shortcuts no longer check for the control key? Why did you take it out?

Comment (rmisio) [inline js/main.js]: Don't merge this. Something is wrong with this branch. It should be 2 files changes (not 30+). I think it needs a merge.

I'll look into it tomorrow.

**Comment by hoffmabc** [comment]
Well I guess we found the missing commits @rmisio

**Comment by hoffmabc** [comment]
I believe the last PR rewrote the git history and removed these later commits, because there was no rebase.

**Comment by rmisio** [comment]
yeah, something weird is going on, almost all these changes were (or look like they were) merged into master on Friday:
https://github.com/OpenBazaar/OpenBazaar-Client/pull/1453

If you look at the other pending PR's, they also have the same issue with a bloated number of files and commits, because they are including what should have already been merged.

Not too familiar with rebase. You think #1453 is the one that screwed up the history?

**Comment by jjeffryes** [comment]
It seems like #1453 got dropped from the commit history. I'm not sure how that could happen, but this commit re-applies the same changes (which is good).

Rebase just means take the current master and apply all your changes to it as if you made all of them after the current state of master, vs. merge which will merge things based on when they were done in each branch. I don't think you want to rebase in this case. Merging should be fine.

**Comment by hoffmabc** [comment]
No I was saying a different merge might have failed to be caught up (through a rebase) and committed a truncated history. Not that you needed to do one @rmisio sorry to be confusing

**Comment by rmisio** [comment]
ok... really, really weird that this happened. Would be nice to know exactly what commit caused it so we could get the details of why it happened and ensure it doesn't happen again.

Anyways, @jjeffryes I adjusted the modified ctrl shortcut code to how it is in master. So, this should be ready for merge.

**Pull request merged**